packages/features/ee/users/components/UnverifiedUsersTable.tsx

```diff
@@ -108,22 +63,13 @@ function UsersTableBare() {
           ...cachedData,
           pages: cachedData.pages.map((page) => ({
             ...page,
-            rows: page.rows.map((row) => {
-              const newUser = row;
-              if (row.id === userId) newUser.locked = locked;
-              return newUser;
-            }),
+            rows: page.rows.filter((row) => row.id !== userId),
```

Comment:
- [X] Mail sent
- [X] Toast shows up
- [ ] Table auto refreshes after user is activated